docs/starter-kit-ui/starter_kit_ui.md

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
can you move this file so that it's nested under "Concepts & Guides"? And then having it the 3rd entry in that list, ie. "Configuring Webservice Properties", then "Configuring Database Properties", then "Configuring the UI."